make/test.mk

```diff
@@ -1,4 +1,18 @@
+COV_DIR = coverage
+GOPKGS = $(shell go list ./... | grep -v /test)
+
 .PHONY: test
 ## runs all tests with bundles assets
 test: generate
-	go test -count=1 ./...
+	# test the test package seperately, no coverage report
+	go test -count=1 ./test
```

Comment:
Also, it's actually fine to include "test" package in the coverage report if it's utility for tests which also covered by tests. Which is the case.
So, let's just run "go test" once for everything. I just thought that "test" package is a utility for tests which doesn't have any tests (to test test utilities) at all.

Reply:
At least now I know how Make expands bash commands :-)

Reply:
Fixed 4b40afb.